src/utils.ts

```diff
@@ -327,5 +327,31 @@ export class Utils {
       }
     }
   }
+
+  /** 
+   * @internal
+   * 
+   * Function used to scroll the page.
+   * 
+   * @param event `MouseEvent` that triggers the resize
+   * @param el `HTMLElement` that's being resized
+   * @param distance Distance to scroll
+   */
+  static updateScrollResize(event: MouseEvent, el: HTMLElement, distance: number): void {
+    const scrollEl = this.getScrollParent(el);
+    const height = scrollEl.clientHeight;
+
+    const top = event.clientY < distance;

```

Comment:
so you're gonna scroll if if mouse is within 'cellHeight' of the edges... might want to document that better, including the param info.

Reply:
Yes, I check if the mouse is a certain distance to the edges (cellHeight) to start scrolling. Also, the closer to the edges the larger the scroll.